dashboard/index.ts

```diff
@@ -41,6 +41,69 @@ async function applyConfigChanges(): Promise<void> {
   }
 }
 
+async function getServiceStatus(): Promise<boolean> {
+  if(Deno.env.get("PIOSK_TEST_MODE") === "true") return true;
```

Comment:
I don't like that `"true"` comes as a string from env.

Must have slipped through the cracks... and should be addressed. If we must use strings, then better to use `PIOSK_MODE=TEST` as the env setup.

**Not needed for this PR. Just chalk it up for later (before v5).**